CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.26)
+
+project(${SKBUILD_PROJECT_NAME}
+    VERSION ${SKBUILD_PROJECT_VERSION}
+    LANGUAGES C
+)
+
+# Path to custom CMake functions
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
+
+# Name of the vendored library CMake target
+include(SetDefaultVendoredLibrary)
+SetDefaultVendoredLibrary()
+
+set(CFFI_INPUT_LIBRARY ${VENDORED_LIBRARY_CMAKE_TARGET})
+
+# Set the output directories for the generated C code and headers
+set(CFFI_C_CODE_DIR ${PROJECT_BINARY_DIR}/_gen_c_file)
+set(CFFI_C_CODE "${CFFI_INPUT_LIBRARY}_cffi_bindings.c")
+set(CFFI_HEADERS_DIR ${PROJECT_BINARY_DIR}/_gen_cffi_headers)
+
+# Shared object that wraps the CFFI binding of the vendored library
+set(CFFI_OUTPUT_LIBRARY "_${VENDORED_LIBRARY_PKG_CONFIG}")
+
+# Setting python for the host system (before change in CMAKE_SYSTEM_PROCESSOR)
+find_package(Python 3 REQUIRED COMPONENTS Interpreter Development.Module Development.SABIModule)
+
+include(SetCrossCompilerGithubActions)
+SetCrossCompilerGithubActions()
+
+# Add the subdirectories. Append CONDA to the PKG_CONFIG_PATH
+if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    set(PKG_CONFIG_ARGN "--dont-define-prefix")
+endif()
+find_package(PkgConfig REQUIRED)
```

Comment:
It is if you have this. :)

Reply:
Okay I had success! I installed https://pypi.org/project/pypkgconf/ which comes with a binary `pkgconf`. I then copied that as `pkg-config` to somewhere on my PATH and a wheel was produced. Is this what Windows users should be doing?

Reply:
If you build wheels with cibuildwheel, then windows users don’t need to do anything. it’s only matters for platforms, not supported by a wheel.